### PR TITLE
Update Chinese Localization

### PR DESCRIPTION
**Pull request by auroursa**
This PR includes the latest translations for zh-CN and zh-TW, and cleans up outdated strings.

### CODE REVIEW

**Comment by KuwaLee** [comment]
@cirx1e

File: src/locale/locales/zh-TW/messages.po [hunk 1]
```diff
@@ -1134,29 +1110,29 @@ msgstr "複製"
 
 #: src/view/com/modals/ChangeHandle.tsx:474
 msgid "Copy {0}"
-msgstr "複製 {0}"
+msgstr "複製{0}"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:1113]:
```suggestion
msgstr "複製 {0}"
```

**Review by cirx1e — COMMENTED** [review]
LGTM

**Review by pfrazee — APPROVED** [review]
Thanks!!

**Pull request merged**